tsconfig.json

```diff
@@ -20,7 +20,7 @@
     "module": "commonjs",
     "lib": ["es6", "dom", "es2016", "esnext.asynciterable"],
     "declaration": true,
-    "outDir": ".",
+    "outDir": "./dist",
```

Comment:
open to ideas regarding naming for this, but it's essentially the compiled `js` and `.d.ts`

Reply:
I like it